### ISSUE TITLE
Anonymous Union Support

### ISSUE THREAD

**Issue by steffengy** [cznic/cc#97]
Currently we have a kinda hackish/suboptimal solution
which should be replaced eventually

Tracking issue for c6e08ea

**Comment by cznic** [comment]
Thanks for filling the issue. I guess anonymous structs probably have the some problem.

**Comment by steffengy** [comment]
Yeah the current implementation just does what's necessary to somehow get those
ugly anonymous union members to work for the windows build :)
